test/acceptance/features/workloadResourceBinding.feature

```diff
@@ -0,0 +1,468 @@
+            """
+        And The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: $scenario_id-backend
+                annotations:
+                    service.binding/host: path={.spec.host}
+                    service.binding/username: path={.spec.username}
+            spec:
+                host: example.common
+                username: foo
+            """
+        And The Workload Resource Mapping is present
+            """
+            apiVersion: servicebinding.io/v1alpha3
+            kind: ClusterWorkloadResourceMapping
+            metadata:
+                name: sampleapps.example.org
+            spec:
+                versions:
+                  - version: "v1"
+                    containers:
+                      - path: .samplespec.template.spec.containers[*]
+                      - path: .samplespec.template.spec.initContainers[*]
+                    volumes: .samplespec.template.spec.volumes
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id-binding
+            spec:
+                services:
+                  - group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo-svc
+                application:
+                    name: demo-cronjob
+                    group: example.org
+                    version: v1
+                    resource: sampleapps
+            """
+        Then Service Binding is ready
+        And Content of file "/bindings/$scenario_id-binding/host" in application pod is
+            """
+            example.common
+            """
+        And Content of file "/bindings/$scenario_id-binding/username" in application pod is
+            """
+            foo
+            """
+
+    Scenario: A workload resource mapping that doesn't specify defaults should use PodSpec-compatible defaults
+        Given The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: $scenario_id-secret
+            stringData:
+                username: AzureDiamond
+                password: hunter2
+                type:     secret
+            """
+        And OLM Operator "not_a_pod" is running
+        And The Workload Resource Mapping is present
+            """
+            apiVersion: servicebinding.io/v1alpha3
+            kind: ClusterWorkloadResourceMapping
+            metadata:
+                name: notpodspecs.stable.example.com
+            spec:
+                versions:
+                  - version: "*"
+                    volumes: .spec.volumeData
+                    containers:
+                      - path: .spec.containerSpecs[]
```

Comment:
Both volume and container paths are given as input. This is against the description of the scenario.
I think we can create two scenarios.
1. The volume is not specified, then it falls back to PoSpec-able default (`.spec.template.spec.volumes`).
2. The containers path is not specified, then it falls back to PoSpec-able default (`.spec.template.spec.containers[*]`)